src/ensembl/utils/database/unittestdb.py

```diff
@@ -66,49 +73,75 @@ class UnitTestDB:
 
     """
 
-    def __init__(self, server_url: StrURL, dump_dir: StrPath, name: Optional[str] = None) -> None:
+    def __init__(
+        self,
+        server_url: StrURL,
+        dump_dir: StrPath | None = None,
+        name: str | None = None,
+        metadata: MetaData | None = None,
+        tmp_path: Path | None = None,
```

Comment:
```suggestion
        tmp_path: StrPath | None = None,
```
There is no intention in this library to stop supporting paths as strings for the time being.